### ISSUE TITLE
Docs are failing

### ISSUE THREAD

**Issue by whitead** [ur-whitelab/hoomd-tf#320]
@RainierBarrett Can you take a look? I don't know enough about the system. 

```
Read the Docs build information
Build id: 14251388
Project: hoomd-tf
Version: latest
Commit: e3bdc41ee3fd609d1573bbb88c8627c850a64492
Date: 2021-07-15T20:27:43.466610Z
State: finished
Success: False


[rtd-command-info] start-time: 2021-07-15T20:27:44.601203Z, end-time: 2021-07-15T20:27:47.388007Z, duration: 2, exit-code: 0
git clone --no-single-branch --depth 50 https://github.com/ur-whitelab/hoomd-tf .
Cloning into '.'...

[rtd-command-info] start-time: 2021-07-15T20:27:47.527807Z, end-time: 2021-07-15T20:27:48.212363Z, duration: 0, exit-code: 0
git checkout --force origin/master
Note: checking out 'origin/master'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at e3bdc41 Benchmarking via pytest (#316)

[rtd-command-info] start-time: 2021-07-15T20:27:48.347306Z, end-time: 2021-07-15T20:27:48.411024Z, duration: 0, exit-code: 0
git clean -d -f -f


[rtd-command-info] start-time: 2021-07-15T20:27:48.536712Z, end-time: 2021-07-15T20:27:48.597608Z, duration: 0, exit-code: 0
git rev-parse HEAD
e3bdc41ee3fd609d1573bbb88c8627c850a64492

[rtd-command-info] start-time: 2021-07-15T20:27:49.860461Z, end-time: 2021-07-15T20:27:53.596946Z, duration: 3, exit-code: 0
python3.7 -mvirtualenv /home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest
Using base prefix '/home/docs/.pyenv/versions/3.7.9'
New python executable in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/bin/python3.7
Also creating executable in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/bin/python
Installing setuptools, pip, wheel...
done.

[rtd-command-info] start-time: 2021-07-15T20:27:53.703800Z, end-time: 2021-07-15T20:27:54.739956Z, duration: 1, exit-code: 0
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/bin/python -m pip install --upgrade --no-cache-dir pip setuptools
Requirement already satisfied: pip in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/lib/python3.7/site-packages (21.1.3)
Requirement already satisfied: setuptools in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/lib/python3.7/site-packages (57.2.0)

[rtd-command-info] start-time: 2021-07-15T20:27:54.824560Z, end-time: 2021-07-15T20:28:02.406503Z, duration: 7, exit-code: 0
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/bin/python -m pip install --upgrade --no-cache-dir mock==1.0.1 pillow==5.4.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<2.2
Collecting mock==1.0.1
  Downloading mock-1.0.1.zip (861 kB)
Collecting pillow==5.4.1
  Downloading Pillow-5.4.1-cp37-cp37m-manylinux1_x86_64.whl (2.0 MB)
Collecting alabaster!=0.7.5,<0.8,>=0.7
  Downloading alabaster-0.7.12-py2.py3-none-any.whl (14 kB)
Collecting commonmark==0.8.1
  Downloading commonmark-0.8.1-py2.py3-none-any.whl (47 kB)
Collecting recommonmark==0.5.0
  Downloading recommonmark-0.5.0-py2.py3-none-any.whl (9.8 kB)
Collecting sphinx<2
  Downloading Sphinx-1.8.5-py2.py3-none-any.whl (3.1 MB)
Collecting sphinx-rtd-theme<0.5
  Downloading sphinx_rtd_theme-0.4.3-py2.py3-none-any.whl (6.4 MB)
Collecting readthedocs-sphinx-ext<2.2
  Downloading readthedocs_sphinx_ext-2.1.4-py2.py3-none-any.whl (29 kB)
Collecting future
  Downloading future-0.18.2.tar.gz (829 kB)
Collecting docutils>=0.11
  Downloading docutils-0.17.1-py2.py3-none-any.whl (575 kB)
Requirement already satisfied: setuptools in /home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/lib/python3.7/site-packages (from sphinx<2) (57.2.0)
Collecting Pygments>=2.0
  Downloading Pygments-2.9.0-py3-none-any.whl (1.0 MB)
Collecting sphinxcontrib-websupport
  Downloading sphinxcontrib_websupport-1.2.4-py2.py3-none-any.whl (39 kB)
Collecting babel!=2.0,>=1.3
  Downloading Babel-2.9.1-py2.py3-none-any.whl (8.8 MB)
Collecting packaging
  Downloading packaging-21.0-py3-none-any.whl (40 kB)
Collecting imagesize
  Downloading imagesize-1.2.0-py2.py3-none-any.whl (4.8 kB)
Collecting requests>=2.0.0
  Downloading requests-2.26.0-py2.py3-none-any.whl (62 kB)
Collecting Jinja2>=2.3
  Downloading Jinja2-3.0.1-py3-none-any.whl (133 kB)
Collecting snowballstemmer>=1.1
  Downloading snowballstemmer-2.1.0-py2.py3-none-any.whl (93 kB)
Collecting six>=1.5
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pytz>=2015.7
  Downloading pytz-2021.1-py2.py3-none-any.whl (510 kB)
Collecting MarkupSafe>=2.0
  Downloading MarkupSafe-2.0.1-cp37-cp37m-manylinux2010_x86_64.whl (31 kB)
Collecting charset-normalizer~=2.0.0
  Downloading charset_normalizer-2.0.2-py3-none-any.whl (35 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2021.5.30-py2.py3-none-any.whl (145 kB)
Collecting idna<4,>=2.5
  Downloading idna-3.2-py3-none-any.whl (59 kB)
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.6-py2.py3-none-any.whl (138 kB)
Collecting pyparsing>=2.0.2
  Downloading pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Collecting sphinxcontrib-serializinghtml
  Downloading sphinxcontrib_serializinghtml-1.1.5-py2.py3-none-any.whl (94 kB)
Building wheels for collected packages: mock, future
  Building wheel for mock (setup.py): started
  Building wheel for mock (setup.py): finished with status 'done'
  Created wheel for mock: filename=mock-1.0.1-py3-none-any.whl size=23772 sha256=4b0c6386d79a5782ca3e895cbe947f9b5f830749d54cee39c0e8ec70c74a0578
  Stored in directory: /tmp/pip-ephem-wheel-cache-hriyy4x7/wheels/7a/94/b1/0fdc5339a5bd487a5cc15421ec200d9ba3e2aa7190e4a727f1
  Building wheel for future (setup.py): started
  Building wheel for future (setup.py): finished with status 'done'
  Created wheel for future: filename=future-0.18.2-py3-none-any.whl size=491070 sha256=aa92687eb10972041888b22f27f65a11574d48cc275808dcd7f6615eeb8dae17
  Stored in directory: /tmp/pip-ephem-wheel-cache-hriyy4x7/wheels/56/b0/fe/4410d17b32f1f0c3cf54cdfb2bc04d7b4b8f4ae377e2229ba0
Successfully built mock future
Installing collected packages: urllib3, sphinxcontrib-serializinghtml, pytz, pyparsing, MarkupSafe, idna, charset-normalizer, certifi, sphinxcontrib-websupport, snowballstemmer, six, requests, Pygments, packaging, Jinja2, imagesize, future, docutils, babel, alabaster, sphinx, commonmark, sphinx-rtd-theme, recommonmark, readthedocs-sphinx-ext, pillow, mock
Successfully installed Jinja2-3.0.1 MarkupSafe-2.0.1 Pygments-2.9.0 alabaster-0.7.12 babel-2.9.1 certifi-2021.5.30 charset-normalizer-2.0.2 commonmark-0.8.1 docutils-0.17.1 future-0.18.2 idna-3.2 imagesize-1.2.0 mock-1.0.1 packaging-21.0 pillow-5.4.1 pyparsing-2.4.7 pytz-2021.1 readthedocs-sphinx-ext-2.1.4 recommonmark-0.5.0 requests-2.26.0 six-1.16.0 snowballstemmer-2.1.0 sphinx-1.8.5 sphinx-rtd-theme-0.4.3 sphinxcontrib-serializinghtml-1.1.5 sphinxcontrib-websupport-1.2.4 urllib3-1.26.6

[rtd-command-info] start-time: 2021-07-15T20:28:02.561887Z, end-time: 2021-07-15T20:28:07.872484Z, duration: 5, exit-code: 1
/home/docs/checkouts/readthedocs.org/user_builds/hoomd-tf/envs/latest/bin/python -m pip install --exists-action=w --no-cache-dir -r requirements.txt
Collecting tensorflow>=2.3
  Downloading tensorflow-2.5.0-cp37-cp37m-manylinux2010_x86_64.whl (454.3 MB)
ERROR: Could not find a version that satisfies the requirement hoomd>=2.6.0 (from versions: none)
ERROR: No matching distribution found for hoomd>=2.6.0

```

**Comment by RainierBarrett** [comment]
This is weird because I thought we had set up mock imports of HOOMD for doc building purposes [here](https://github.com/ur-whitelab/hoomd-tf/blob/e3bdc41ee3fd609d1573bbb88c8627c850a64492/sphinx-docs/source/conf.py#L78). Is this just from the readthedocs.io output? Anyone able to build locally?

**Comment by whitead** [comment]
I was able to build locally, but had to [make changes](https://github.com/ur-whitelab/hoomd-tf/pull/319/commits/5dbc2d134333178cacede6039ec2ae88d47eea38) that indicated it wasn't correctly mocking.

**Comment by RainierBarrett** [comment]
I think I might have an idea why... It says on the [config file](https://docs.readthedocs.io/en/stable/config-file/v2.html) docs page that the config file must be named exactly `.readthedocs.yml` but it looks like ours is just `readthedocx.yml`

**Comment by whitead** [comment]
Got it. Will try that.

**Comment by whitead** [comment]
Fixed via #321
